EssentialApp/EssentialAppTests/ImageCommentsUIIntegrationTests.swift

```diff
@@ -0,0 +1,149 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import UIKit
+import EssentialApp
+@testable import EssentialFeed
```

Comment:
Following from my previous comment, we won't need `@testable` by declaring a public initializer on `ImageComment`:
```suggestion
import EssentialFeed
```